munit/shared/src/main/scala/munit/FileLayout.scala

```diff
@@ -0,0 +1,75 @@
+package munit
+
+import java.nio.charset.Charset
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.StandardOpenOption
+import java.nio.file.Path
+
+trait FileLayout {
+
+  protected def fileLayoutDefaultTempDirName = "file-layout"
+
+  /**
+   * Parse a file layout from string
+   *
+   * @param layout
+   * @return the file layout as a map of path -> content
+   * @throws IllegalArgumentException if the layout is formally invalid
+   */
+  protected def parseFileLayout(layout: String): Map[String, String] =
+    if (!layout.trim.isEmpty) {
+      val lines = layout.replaceAllLiterally("\r\n", "\n")
+      lines
+      // lines starting with '/' are file paths, but lines starting with '//' are comments
+        .split("(\n/[^/])")
```

Comment:
here's where I've changed the regex.
It was 

```suggestion
        .split("(?=\n/[^/])")
```

before, but Scala Native implementation does not support lookaheads.

I've adjusted line 27 to not remove the leading "\n" (since it's dropped by `split` now) and it seems to work as expected